README.md

```diff
@@ -170,23 +172,24 @@ Additionally, required for tests:
 
 ### Memory pool managers
 
-#### proxy_pool (part of libumf)
+#### Proxy pool (part of libumf)
 
 This memory pool is distributed as part of libumf. It forwards all requests to the underlying
 memory provider. Currently umfPoolRealloc, umfPoolCalloc and umfPoolMallocUsableSize functions
 are not supported by the proxy pool.
 
-#### libumf_pool_disjoint
+#### Disjoint pool
 
 TODO: Add a description
 
 ##### Requirements
 
 To enable this feature, the `UMF_BUILD_LIBUMF_POOL_DISJOINT` option needs to be turned `ON`.
 
-#### libumf_pool_jemalloc
+#### Jemaloc pool
 
-libumf_pool_jemalloc is a [jemalloc](https://github.com/jemalloc/jemalloc)-based memory pool manager built as a separate static library.
+Jemaloc pool is a [jemalloc](https://github.com/jemalloc/jemalloc)-based memory 
+pool manager built as a separate static library libumf_pool_jemalloc.
```

Comment:
On Windows it has a different name. You could ie. write the whole linux name like `libjemalloc_pool.a` and in parenthesis the Windows name - `jemalloc_pool.lib`

Reply:
done